AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_BasicTests.py

```diff
@@ -213,13 +213,13 @@ def MaterialEditor_BasicFunctionalityChecks_AllChecksPass():
         Report.result(Tests.viewport_background_selected, neutral_urban_background_loaded is True)
 
         # 11. Verify the lighting background asset is the expected value.
-        neutral_urban_background_path = os.path.join(
-            azlmbr.paths.engroot, "Gems", "Atom", "Tools", "MaterialEditor", "Assets", "MaterialEditor",
-            "LightingPresets", "neutral_urban.lightingpreset.azasset")
-        neutral_urban_background_path = neutral_urban_background_path.replace(os.sep, '/')
-        Report.result(
-            Tests.viewport_background_has_expected_asset,
-            atom_tools_utils.get_last_lighting_preset_path() == neutral_urban_background_path)
+        # neutral_urban_background_path = os.path.join(
+        #     azlmbr.paths.engroot, "Gems", "Atom", "Tools", "MaterialEditor", "Assets", "MaterialEditor",
+        #     "LightingPresets", "neutral_urban.lightingpreset.azasset")
+        # neutral_urban_background_path = neutral_urban_background_path.replace(os.sep, '/')
+        # Report.result(
+        #     Tests.viewport_background_has_expected_asset,
+        #     atom_tools_utils.get_last_lighting_preset_path() == neutral_urban_background_path)
```

Comment:
Can we add the GHI tracking ticket for this disabled code?

Reply:
I had a JIRA created but here is a GHI to match it: https://github.com/o3de/o3de/issues/13354